generic-sensor/generic-sensor-tests.js

```diff
@@ -32,6 +32,40 @@ function reading_to_array(sensor) {
   return arr;
 }
 
+function load_iframe(src) {
+  return new Promise(resolve => {
+    const iframe = document.createElement("iframe");
+    iframe.onload = () => { resolve(iframe); };
+    iframe.srcdoc = src;
+    iframe.style.display = "none";
+    document.documentElement.appendChild(iframe);
+  });
+}
+
+function wait_for_message(iframe) {
+  return new Promise(resolve => {
+    self.addEventListener("message", function listener(e) {
+      if (e.source === iframe.contentWindow) {
+        resolve(e.data);
+        self.removeEventListener("message", listener);
+      }
+    });
+  });
+}
+
+function make_script(sensorType) {
```

Comment:
Could the name of this function be more descriptive? tryToCreateSensorObject or smth else?